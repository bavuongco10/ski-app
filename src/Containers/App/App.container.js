import React from 'react';
import { Route } from 'react-router-dom';

import Home from '../Home/Home.container';
import HeaderReact from '../../Components/HeaderReact/HeaderReact.component';

const App = () => (
  <div className="App">
    <HeaderReact />
    <main>
      <Route exact path="/" component={Home} />
    </main>
  </div>
);
export default App;
