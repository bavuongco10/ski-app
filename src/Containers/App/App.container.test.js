import React from 'react';

import App from './App.container';

it('renders without crashing', () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
