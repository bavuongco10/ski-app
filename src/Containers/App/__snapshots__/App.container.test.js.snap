// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<div
  className="App"
>
  <HeaderReact />
  <main>
    <Route
      component={[Function]}
      exact={true}
      path="/"
    />
  </main>
</div>
`;
